crates/util/python-utils/Cargo.toml

```diff
@@ -13,8 +13,10 @@ name = "strata_utils"
 crate-type = ["cdylib"]
 
 [dependencies]
-shrex = { version = "0.1.0", path = "../shrex", features = ["serde"] }
+strata-primitives.workspace = true
+strata-crypto.workspace = true
 
+shrex = { version = "0.1.0", path = "../shrex", features = ["serde"] }
```

Comment:
shree is not "external". Should be moved up and globbed together with the `strata-*` crates.